larpandoracontent/LArControlFlow/NeutrinoIdTool.cc

```diff
@@ -167,6 +187,8 @@ unsigned int NeutrinoIdTool::CountNeutrinoInducedHits(const CaloHitList &hitList
     return nNuHits;
 }
 
+
```

Comment:
Two empty lines! Two!

Reply:
I thought it helped the fung shui of the code, but happy to remove.